src/main/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationService.scala

```diff
@@ -42,7 +42,8 @@ trait MethodConfigurationService extends HttpService with PerRequestCreator with
               val extReq = Post(FireCloudConfig.Rawls.copyFromMethodRepoConfigUrl, copyMethodConfig)
               externalHttpPerRequest(requestContext, extReq)
             }
-          }
+          } ~
+          complete(StatusCodes.MethodNotAllowed, ErrorReport(StatusCodes.MethodNotAllowed, "HTTP method not allowed"))
```

Comment:
I don't think we have anything - I only know about it from the documentation: http://spray.io/documentation/1.2.2/spray-routing/key-concepts/rejections/#rejectionhandler